tasmota/my_user_config.h

```diff
@@ -738,6 +738,7 @@
 //    #define DS3231_ENABLE_TEMP                   //   In DS3231 driver, enable the internal temperature sensor
 //    #define USE_BM8563                           // [I2cDriver59] Enable BM8563 RTC - found in M5Stack - support both I2C buses on ESP32 (I2C address 0x51) (+2.5k code)
 //    #define USE_PCF85363                         // [I2cDriver66] Enable PCF85363 RTC - found Shelly 3EM (I2C address 0x51) (+0k7 code)
+    #define USE_AMSX915                          // [I2CDriver86] Enable AMS5915/AMS6915 pressure/temperature sensor (+1k2 code)
```

Comment:
Not sure it should be enabled by default. @Jason2866 @arendst

Reply:
You're right but in the end it needs to be disabled before merge so pls comment this line and I'll merge.